@here/harp-mapview/lib/Tile.ts

```diff
@@ -256,6 +257,46 @@ interface PolygonFadingParameters extends FadingParameters {
     lineFadeFar?: number;
 }
 
+/**
+ * Proxies a Tile with a given offset (a numeric value to represent the longitudinal shift of the
+ * tile (a multiple of 360 degrees)). The given offset should be non zero, because we don't need to
+ * proxy the Tile at offset 0 (a Tile is by definition at offset 0).
+ *
+ * The TileObject array and the TextElement array are cloned and stored in this object.
+ */
+export class TileProxy {
```

Comment:
I concur with @StefanDachwitz comment and think that proxy is not needed at all.
All we need is a proper tile interface (`ITile` and two implementation
 - old one , that owns whole geometry and has `offset` = 0
 - proxy, that 
    - clones geometry of original tile when it's updated
    - have`offset != 0`
    - carefully disposes itself, so it doesn't dispose parent tile buffers and materials

Actually, it might be reasonable to refererence-count buffers and materials.

Reply:
@zbigg, I spoke with @StefanDachwitz after the Berlin standup.

I will look into see how much effort such an interface could be. It will reduce the mental effort required for sure, it could however be a lot of work to implement the interface for the proxy. Will investigate however.

Reply:
@zbigg , @StefanDachwitz , I have implemented the ITile interface and two concrete sub tiles: Tile and TileProxy, please review at your next convenience.